examples/two-step-transfer/Cargo.toml

```diff
@@ -10,12 +10,10 @@ repository = "https://github.com/concordium/concordium-rust-smart-contracts"
 readme = "../README.md"
 
 [features]
-default = ["std"]
+default = ["std", "concordium-quickcheck"]
 
 std = ["concordium-std/std"]
-
-[dependencies]
-byteorder = "1.3"
+concordium-quickcheck = ["concordium-std/concordium-quickcheck", "concordium-contracts-common/concordium-quickcheck"]
```

Comment:
Instead of enabling `"concordium-contracts-common/concordium-quickcheck"` here, it should be done in `concordium-std`

Reply:
Done.